### ISSUE TITLE
documentation error for plot: x expects results, not number of qsorts in study

### ISSUE THREAD

**Issue by maxheld83** [aiorazabala/qmethod#17]
I fixed this myself in the dev channel w/o a pull request; hope I got it right and it's ok.

https://github.com/aiorazabala/qmethod/commit/8fa039f05f99a922505dc005c4684948f06c818e

**Comment by maxheld83** [comment]
this can be closed if #37 is merged, I think

**Comment by aiorazabala** [comment]
I can't understand the issue from the title.

**Comment by maxheld83** [comment]
sorry, @aiorazabala , that was bad issue logging on my part. Almost couldn't understand my own issue title :).

The documentation for `plot` in `plot.QmethodRes.Rd` reads:

```
\arguments{
  \item{x}{number of Q-sorts in the study.}
```

I think that is incorrect, and `plot` actually expects:

```
 \item{x}{results object returned by \code{\link{qmethod}}.}
```

Did I get that right?
So it's a minor thing, anyway.

I fixed this myself (without pull request) on `dev` in this commit: https://github.com/aiorazabala/qmethod/commit/8fa039f05f99a922505dc005c4684948f06c818e

Hope I got it right.
